src/igvfd/searches/configs/signal_file.py

```diff
@@ -22,6 +22,72 @@ def signal_file():
             'lab': {
                 'title': 'Lab'
             },
+            'status': {
+                'title': 'Status'
+            },
+            'strand_specificity': {
+                'title': 'Strand Specificity'
+            },
+            'collections': {
+                'title': 'Collections'
+            },
```

Comment:
Your lab.title here is preferred, please remove duplicate lab in L22

Reply:
done